wordpress-mobile/WordPress-Android
# Enabling Recommend the app deature.

This PR enables the #15269 in production

Depends on #15270 and #15277 that must be merged first.

## To test
- Remove the app if already installed
- Use the jalapeno apk from this CI to install the WordPress app and check the confirm the share menu in Me screen IS available by default 
- Use the jalapeno apk from this CI to install the Jetpack app and confirm the share menu in Me screen IS NOT available by default
- Smoke test the feature (you can refer to the PR description in #15270 for more details if needed)

## Regression Notes
1. Potential unintended areas of impact
None expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None expected

3. What automated tests I added (or what prevented me from doing so)
Added unit test for the feature and tracking

PR submission checklist:
